example/client/main.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	rnp "github.com/Autodoc-Technology/redis-nats-proxy"
+	"github.com/nats-io/nats.go"
+	"github.com/redis/go-redis/v9"
+	"log/slog"
+	"net"
+	"os"
+	"os/signal"
+	"sync"
+)
+
+func main() {
+	log := slog.New(slog.NewTextHandler(os.Stderr, &slog.HandlerOptions{Level: slog.LevelDebug}))
+	slog.SetDefault(log)
+
+	ctx, stop := signal.NotifyContext(context.Background(), os.Interrupt, os.Kill)
+	defer stop()
+
+	nc, err := nats.Connect(nats.DefaultURL)
+	if err != nil {
+		slog.Error("connect to nats", "err", err)
+		return
+	}
+	defer nc.Close()
+
+	// read first argument as the address of the Redis server
+	// if no argument is provided, use the default address
+	addr := "localhost:6379"
+	if len(os.Args) > 1 {
+		addr = os.Args[1]
```

Comment:
Seems we never check how many args we have

Reply:
ugu